### PR TITLE
mesheryctl Fix `system token create` unhelpful error message.

### LINKED ISSUE #4214: [mesheryctl] `system token create`shows unhelpful error when token name is missing
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior

`mesheryctl system token create` is used to create a new token in the meshconfig(.meshery/config.yaml) file. 

When the token name is not provided, it shows an unhelpful error:

Screenshot 2021-09-20 at 2 59 42 PM

#### Desired Behavior

Mention that the token name is needed instead of just saying there is a missing argument.

#### Environment

- Host OS: Mac
- Meshery Server Version: edge-latest
- Meshery Client Version: edge-latest
- Platform: Docker

---

#### Contributor Resources
- [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Zanda256**
**Description**

This PR fixes #4214 

**Notes for Reviewers**
I created a function that mimics cobra.ExactArgs() but returns an informative error message when no token name or an empty string is supplied by the user.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -921,6 +927,7 @@ github.com/mitchellh/mapstructure v1.3.2/go.mod h1:bFUtVrKA4DC2yAKiSyO/QUcy7e+RR
 github.com/mitchellh/mapstructure v1.3.3/go.mod h1:bFUtVrKA4DC2yAKiSyO/QUcy7e+RRV2QTWOzhPopBRo=
 github.com/mitchellh/mapstructure v1.4.1 h1:CpVNEelQCZBooIPDn+AR3NpivK/TIKU8bDxdASFVQag=
 github.com/mitchellh/mapstructure v1.4.1/go.mod h1:bFUtVrKA4DC2yAKiSyO/QUcy7e+RRV2QTWOzhPopBRo=
+github.com/mitchellh/osext v0.0.0-20151018003038-5e2d6d41470f h1:2+myh5ml7lgEU/51gbeLHfKGNfgEQQIWrlbdaOsidbQ=
```
Comment (leecalcote) [inline go.sum:930]: From a security perspective, each of these changes should be ok as these are not net new packages, but updates to each.

**Comment by Zanda256** [comment]
@leecalcote ,Sure I'll work on it ASAP.
Also , as @navendu-pottekkat had suggested earlier, should I check the rest of the "system token " commands for this same issue?

**Comment by leecalcote** [comment]
@Zanda256, yes, that would be great. 👍

**Comment by piyushsingariya** [comment]
Please reset the lock.json files too.

**Pull request merged**